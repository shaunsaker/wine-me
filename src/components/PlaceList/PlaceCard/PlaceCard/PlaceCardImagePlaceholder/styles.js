import { StyleSheet } from 'react-native';

import styleConstants from '../../../../../styleConstants';

const styles = StyleSheet.create({
  container: {
    // from RemoteImage
    flex: 1,
    height: 200,
    backgroundColor: styleConstants.colors.primary,
    borderRadius: styleConstants.dimensions.borderRadius,
    justifyContent: 'center',
    alignItems: 'center',
  },
  icon: {
    fontSize: 36,
    color: styleConstants.colors.white,
  },
});

export default styles;
