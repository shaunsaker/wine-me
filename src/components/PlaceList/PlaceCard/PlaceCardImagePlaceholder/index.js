import React from 'react';
import { View } from 'react-native';
import Icon from 'react-native-vector-icons/MaterialIcons';

import styles from './styles';

const propTypes = {};

const defaultProps = {};

const PlaceCardImagePlaceholder = () => {
  return (
    <View style={styles.container}>
      <Icon name="info" style={styles.icon} />
    </View>
  );
};

PlaceCardImagePlaceholder.propTypes = propTypes;
PlaceCardImagePlaceholder.defaultProps = defaultProps;

export default PlaceCardImagePlaceholder;
