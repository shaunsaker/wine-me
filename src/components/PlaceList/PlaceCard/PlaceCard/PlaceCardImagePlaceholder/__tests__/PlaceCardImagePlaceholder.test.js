import React from 'react';
import renderer from 'react-test-renderer';

import PlaceCardImagePlaceholder from '..';

describe('PlaceCardImagePlaceholder', () => {
  describe('renders', () => {
    it('renders', () => {
      const component = renderer.create(<PlaceCardImagePlaceholder />);

      expect(component).toMatchSnapshot();
    });
  });
});
