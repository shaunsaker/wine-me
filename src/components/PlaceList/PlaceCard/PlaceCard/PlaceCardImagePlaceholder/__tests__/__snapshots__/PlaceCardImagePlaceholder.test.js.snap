// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceCardImagePlaceholder renders renders 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#3D84A8",
      "borderRadius": 5,
      "flex": 1,
      "height": 200,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "#FFFFFF",
          "fontSize": 36,
        },
        Object {
          "fontFamily": "Material Icons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    
  </Text>
</View>
`;
